src/vivarium_csu_ltbi/data/gbd2019_hh_tb_model.py

```diff
@@ -0,0 +1,148 @@
+import numpy as np
+import pandas as pd
+from get_draws.api import get_draws
+from vivarium.interpolation import Interpolation
+
+master_dir = '/home/j/Project/simulation_science/latent_tuberculosis_infection/literature/household_structure/microdata/'

```

Comment:
We don't want file paths in our public facing repos.

Reply:
I've seen them in many other repos, e.g. https://github.com/ihmeuw/vivarium_csu_swissre_breast_cancer/blob/master/src/vivarium_csu_swissre_breast_cancer/paths.py#L7; why don't we want them?

Reply:
> We don't want file paths in our public facing repos.

We have `global_path` for the household microdata.